apps/site/components/Common/Partners/PartnerLogo/index.module.css

```
@@ -0,0 +1,19 @@
+@reference "../../../../styles/index.css";
+
+.partnerIcon {
+  @apply flex
+    h-[114px]
+    max-h-[114px]
```

```suggestion
    h-28
    max-h-28
```

If possible, instead of specifying pixel values directly, it's better to use a Tailwind utility class like `h-28`, which corresponds to `112px` / `7rem`